content/en/guide/v10/typescript.md

```diff
@@ -144,47 +129,43 @@ type ExpandableState = {
 };
 
 
-// Bind generics to ExpandableProps and ExpandableState
+// ExpandableProps and ExpandableState are passed as generic type parameter.
 class Expandable extends Component<ExpandableProps, ExpandableState> {
   constructor(props: ExpandableProps) {
     super(props);
-    // this.state is an object with a boolean field `toggle`
-    // due to ExpandableState
+    // `this.state` is a object with `toggle` property which is boolean type by ExpandableState.
     this.state = {
       toggled: false
     };
   }
-  // `this.props.title` is string due to ExpandableProps
   render() {
     return (
       <div class="expandable">
         <h2>
+          // `this.props.title` is string type by ExpandableProps.
           {this.props.title}{" "}
-          <button
-            onClick={() => this.setState({ toggled: !this.state.toggled })}
-          >
+          <button onClick={() => this.setState({ toggled: !this.state.toggled })}>
             Toggle
           </button>
         </h2>
+        // `this.props.children` is `ComponentChildren` type by default.
         <div hidden={this.state.toggled}>{this.props.children}</div>
       </div>
     );
   }
 }
 ```
 
-Class components include children by default, typed as `ComponentChildren`.
-
-## Typing events
```

Comment:
I think it is necessary to describe about `TargetedEvent`, since `TargetedEvent` does not exist in React.